.rubocop.yml

```diff
@@ -1081,18 +1081,6 @@ Performance/MapCompact:
 Performance/MethodObjectAsBlock:
   Enabled: false
 
-# TODO: OLD RUBIES - Requires 2.4
```

Comment:
TODOs that required a Ruby version 2.4+ were moved into `rubocop_todo` to revisit later as a team.

Reply:
Ticket: https://github.com/newrelic/newrelic-ruby-agent/issues/1787